package.json

```diff
@@ -23,8 +23,8 @@
     "watch:icons": "yarn workspace @twilio-paste/icons watch",
     "prebuild": "node ./tools/build/pre-build.js && yarn packages:check",
     "build": "lerna run build --ignore @twilio-paste/website --ignore @twilio-paste/theme-designer --ignore @twilio-paste/nextjs-template",
-    "build:js": "yarn prebuild && lerna run build:js --since HEAD --ignore @twilio-paste/website --ignore @twilio-paste/theme-designer",
-    "build:dev": "yarn prebuild && lerna run build:dev --since HEAD --ignore @twilio-paste/website",
+    "build:js": "yarn prebuild && lerna run build:js --ignore @twilio-paste/website --ignore @twilio-paste/theme-designer",
+    "build:dev": "yarn prebuild && lerna run build:dev --ignore @twilio-paste/website",
```

Comment:
Does anyone use this one?

Reply:
Nah, let's remove it. I removed it everywhere except missed this one.